kvm/v12.1.0/README.md

```diff
@@ -0,0 +1,33 @@
+## :zap:  Giant Swarm Release 12.1.0 for KVM :zap:
+
+In this release:
```

Comment:
```suggestion
**This release includes two significant improvements to NGINX Ingress Controller. It also adds support for image registry mirrors, improving reliability.**
```

Just to give a top level summary.

Reply:
@sslavic @snizhana-dynnyk I think the registry mirrors is a pretty significant change and we should call out in the main paragraph. Is this wrong? :D

Reply:
Should we use the language used in the AWS release? https://github.com/giantswarm/releases/tree/master/aws/v11.5.0

Reply:
yes, but without mentioning China